core/src/main/java/ai/timefold/solver/core/impl/domain/variable/declarative/AffectedEntitiesUpdater.java

```diff
@@ -90,30 +81,6 @@ private void initializeChangeQueue(BitSet changed) {
         changed.clear();
     }
 
-    private void updateLoopedStatusOfAffectedEntity(Object affectedEntity) {
```

Comment:
Because we no longer defer the updates (see below), this is no longer necessary.